src/inc/msquicp.h

```diff
@@ -116,6 +116,7 @@ typedef struct QUIC_PRIVATE_TRANSPORT_PARAMETER {
 #endif
 #define QUIC_PARAM_GLOBAL_IN_USE                        0x81000004  // BOOLEAN
 #define QUIC_PARAM_GLOBAL_DATAPATH_FEATURES             0x81000005  // uint32_t
+#define QUIC_PARAM_GLOBAL_GET_DATAPATH_WORKER_POOL      0x81000006  // CXPLAT_WORKER_POOL**
```

Comment:
Don't put `GET` in the name. We just won't allow SET, which is what we do for several parameters.

Reply:
And let's call it `_PLATFORM_WORKER_POOL`.

Reply:
Why return a `**` instead of `*`? Is the problem that it's not necessarily initialized yet? Perhaps trigger lazy instantiation if called?

Reply:
Yeah, it's not a **, its just a *. Doesn't actually change how the code works (A ** has sizeof() equal to a *).

I'll fix the other things.